### PR TITLE
genealotree:0.1.0

### PR DESCRIPTION
**Pull request by DrLoiseau**
I am submitting
- [x] a new package
- [ ] an update for a package

This package is based on CeTZ and it provides functions to draw genealogical trees. It is oriented towards medical genealogy to study genetic disorders inheritance, but you might be able to use it to draw your family tree.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/genealotree/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+version = "0.1.0"
+entrypoint = "genealotree.typ"
+authors = ["DrLoiseau"]
+license = "GPL-3.0-only"
+description = "A package to draw genealogical trees, based on CeTZ"
+repository = "https://codeberg.org/drloiseau/genealogy"
+keywords = ["draw", "cetz", "genealogical", "genealogy", "tree"]
+categories = ["visualization"]
+disciplines = ["anthropology", "biology", "history", "medicine"]
+compiler = "0.10.0"
```
Comment (elegaanz) [inline packages/preview/genealotree/0.1.0/typst.toml:12]: The example directory is generally excluded, as it is not that useful to have it in the local package cache.
```suggestion
compiler = "0.10.0"
exclude = [ "examples" ]
```

File: packages/preview/genealotree/0.1.0/examples/example.typ [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#import "../genealotree.typ": *
```
Comment (elegaanz) [inline packages/preview/genealotree/0.1.0/examples/example.typ:1]: To make this example easier to copy/paste and use as a starting point, using the package specification would be nice.
```suggestion
#import "@preview/genealotree:0.1.0": *
```

File: packages/preview/genealotree/0.1.0/examples/full_example.typ [hunk 3]
```diff
@@ -0,0 +1,84 @@
+#import "../genealotree.typ": genealogy-init, add-phenos, add-person, add-persons, add-union, add-unions, draw-tree
```
Comment (elegaanz) [inline packages/preview/genealotree/0.1.0/examples/full_example.typ:1]: Same remark here.
```suggestion
#import "@preview/genealotree:0.1.0": genealogy-init, add-phenos, add-person, add-persons, add-union, add-unions, draw-tree
```

**Comment by reknih** [comment]
Hey @DrLoiseau! Just wanted to make sure this is still on your radar, comment below if you have any questions about the review.

**Comment by DrLoiseau** [comment]
Yes it is, I planned to add the changes next week, but I hadn't noticed I could just commit the changes from the discussion. Is there anything else to modify ?

**Comment by elegaanz** [comment]
No everything looks good now, thank you!

**Pull request merged**